nix/deps/gradle/proj.list

```diff
@@ -30,7 +30,7 @@ react-native-reanimated
 react-native-safe-area-context
 react-native-shake
 react-native-share
-react-native-splash-screen
+react-native-lottie-splash-screen
```

Comment:
Did you updated this file manually?
Please use `make nix-update-gradle`. It will update nix dependncies, and also most probably will fix android build error.
```
[2022-07-24T13:42:14.808Z] > Could not resolve all artifacts for configuration ':app:prRuntimeClasspath'.
[2022-07-24T13:42:14.808Z]    > Could not resolve com.airbnb.android:lottie:3.4.4.
```

Reply:
Hi, just added a new commit with nix update gradle and pods.

Reply:
The local build passed once I changed the signing (debug) to auto in xcode